modules/watch.cpp

```diff
@@ -377,10 +357,14 @@ class CWatcherMod : public CModule {
                                       "!watch@znc.in PRIVMSG " +
                                       pNetwork->GetCurNick() + " :" + sMessage);
                 } else {
-                    m_Buffer.AddLine(
+                    CQuery* pQuery = pNetwork->AddQuery(WatchEntry.GetTarget());
+                    if (pQuery) {
+                        
+                        pQuery->AddBuffer(
                         ":" + _NAMEDFMT(WatchEntry.GetTarget()) +
```

Comment:
clang-format would make this nicer

Reply:
I ran clang format on this part and got no changes compared to what is commited. Or maybe I do not understand you comment :)

Reply:
Strange, it should have reindented it so that parameters are to the right of the function call.

Reply:
you mean like this?

Reply:
No, but it's better than no offset at all. Are you sure you've run it?

Reply:
yes, it's running on my znc right now.

Reply:
Sorry, by "it" I meant clang-format. I doubt you're running clang-format on your ZNC right now :)

Reply:
haha 🤦‍♂